document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/mapping/smallfull/model/SmallFullApiData.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model;
+
+import com.google.gson.Gson;
+import uk.gov.companieshouse.api.model.accounts.smallfull.CurrentPeriodApi;
+import uk.gov.companieshouse.api.model.accounts.smallfull.PreviousPeriodApi;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+import uk.gov.companieshouse.document.generator.accounts.mapping.smallfull.model.ixbrl.period.Period;
+
+import java.util.Objects;
+
+public class SmallFullApiData {
+
+    private CurrentPeriodApi currentPeriodApi;
+
+    private PreviousPeriodApi previousPeriodApi;
+
+    private CompanyProfileApi companyProfileApi;
+
+    private Period period;
+
+    public CurrentPeriodApi getCurrentPeriodApi() {
+        return currentPeriodApi;
+    }
+
+    public void setCurrentPeriodApi(CurrentPeriodApi currentPeriodApi) {
+        this.currentPeriodApi = currentPeriodApi;
+    }
+
+    public PreviousPeriodApi getPreviousPeriodApi() {
+        return previousPeriodApi;
+    }
+
+    public void setPreviousPeriodApi(PreviousPeriodApi previousPeriodApi) {
+        this.previousPeriodApi = previousPeriodApi;
+    }
+
+    public CompanyProfileApi getCompanyProfileApi() {
+        return companyProfileApi;
+    }
+
+    public void setCompanyProfileApi(CompanyProfileApi companyProfileApi) {
+        this.companyProfileApi = companyProfileApi;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
could we add curly braces as per our coding standards?

Reply:
auto generator equals and hash through intelliJ